### ISSUE TITLE
[Utility Suggestion] Powered Spawner

### ISSUE THREAD

**Issue by EmosewaPixel** [GregTechCE/GregTech#496]
You need to kill a mob while having a DNA Collector in your inventory (might end up being a different item). You put it in the Spawner and at the cost of 4 amps spawns that mob nearby. Spawn area is configurable. Speed and max area increases with tier.

**Comment by Agrax** [comment]
It would less tps-heavy if it used energy and spawned item drops when supplied with a sword (affected by looting n’stuff)

**Comment by idcppl** [comment]
I don't think it needs to be affected by enchantments, since it will be heavily affected by overclocking.
